discover/portal/articles/07-collaborating/01-publishing-blogs.markdown

```diff
@@ -0,0 +1,444 @@
+the first row and/or column as table headers. The controls also enable you to
+add rows, columns, and cells. 
+
+Now you're familiar with the editor's regular mode. If you'd rather work with
+the content's HTML code, you can. To switch the editor to source view, select
+the *Source* icon (`</>`). Note that an icon (`abc`) for regular mode appears,
+giving you the option to switch back to regular view. Source view offers slick
+features allow you to make the editor comfortable and practical. To satisfty
+your eyes, source view's moon icon and sun icon let you switch between night
+mode, a dark background with white type, and day mode, a white background with
+black type. And the built-in syntax coloring helps you identify HTML elements,
+no matter if you're using day or night mode. 
+
+Have you ever cleared off your workspace to continue your work in a clean, fresh
+environment? The editor's fullscreen mode allows you to do the same thing as you
+blog. You can switch to fullscreen mode by selecting the *Fullscreen* icon
+(bidirectional diagonally-crossed arrows ). The *Edit Content* window appears,
+taking up the entire browser screen. The first icons above the editor allow you
+to arrange the HTML editor and rendered content area, horizontally or
+vertically. These dual views allow you to edit the HTML source and render it in
+real time. The last icon at the top of the content editor lets you toggle hiding
+the content rendering area, so you can use the entire browser space to work on
+the HTML code. Fullscreen view can really help you stay "in the zone" as you
+create awesome blog entries. 
+ 
+![Figure 2: Fullscreen view helps you minimize screen clutter and render HTML changes in real time.](../../images/blogs-full-screen-editor.png)
+
+Every 25 seconds, the entry you're editing is automatically saved as a draft; so
+a browser crash or network interruption won't cause you to lose your entry. You
+can exit the fullscreen view, by clicking on *Done* (which saves your content)
+or clicking on *Cancel* to abandon any changes since the last auto-save. From
+the normal sized source view, you can click the `abc` icon to switch back to
+regular view to use the WYSIWYG editor. 
+
+You've learned all the ins and outs of the content editor. Next, you'll learn
+how to specify your blog entry's other characteristics. 
+
+The edit screen's first input field--*Drag \& Drop to Upload*--lets you add a
+cover image (optional) for your entry. This might be an image that represents
+your entry's purpose and attract readers. You can drag and drop onto this field
+any image you like. As an alternative to dragging and dropping an image, you
+click on the *Select File* button to use the image selector to choose an
+existing image in the blog, an image file from Documents and Media, or an image
+outside the portal. You can browse to a file and upload it, or you can specify
+an image's URL for the image selector to take in the image automatically. After
```

Comment:
Regarding the Cover Image I think that right now we don't support specifying a image via URL. It's something that we have in mind but we didn't have the time to do it yet, so I'd skip that for now.